Fastfile

```diff
@@ -268,7 +246,7 @@ reflected in `Info.plist` during building."
 
     pem(
       generate_p12: true,
-      development: match_types[build_configuration] == "development",
+      development: get_match_type(build_configuration: build_configuration) == "development",
```

Comment:
This "development" should be replaced by `GetMatchTypeAction .MATCH_TYPES[:test]`, right?

Reply:
`development` is a `Bool` type. This comparison is right.

Reply:
I like it too 🎉 👍